src/Shared/Infrastructure/EventListener/CacheInvalidationDoctrineEventListener.php

```
@@ -0,0 +1,165 @@
+
+        $unitOfWork = $args->getDocumentManager()->getUnitOfWork();
+
+        $this->queueCreatedDocuments($unitOfWork);
+        $this->queueUpdatedDocuments($unitOfWork);
+        $this->queueDeletedDocuments($unitOfWork);
+    }
+
+    /**
+     * @psalm-suppress UnusedParam Signature required by Doctrine ODM.
+     */
+    public function postFlush(PostFlushEventArgs $args): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)